helpers/dbload.ts

```diff
@@ -96,6 +99,34 @@ UserLesson.belongsTo(User, {
 
 sequelize.sync({ alter: !!process.env.ALTER_DB })
 
+/* istanbul ignore next */
```

Comment:
This will make jest ignore this branch when calculating total coverage, I couldn't figure out how to force istanbul to properly cover it.
If anyone knows how to make this work properly, please comment.

Reply:
We currently ignore files we don't want to cover by adding them into the package.json file:

https://github.com/garageScript/c0d3-app/blob/master/package.json#L120